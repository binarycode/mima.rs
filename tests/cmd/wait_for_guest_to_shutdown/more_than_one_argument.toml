args = 'wait-for-guest-to-shutdown one two'
status.code = 2
