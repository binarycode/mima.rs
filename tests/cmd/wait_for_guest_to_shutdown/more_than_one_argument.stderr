error: unexpected argument 'two' found

Usage: mima wait-for-guest-to-shutdown <GUEST_ID>

For more information, try '--help'.
